recipe/meta.yaml

```diff
@@ -5,6 +5,8 @@
 {% set sha256 = "1fd90354b9cf19232e8f168faf2220e79be555df3aa743242700879e8fd329ee" %}
 
 {% set llvm_variant = os.environ.get('LLVM_VARIANT', 'default') %}
+{% set build_number = "2" %}
+{% set build_string = "{}_{}".format(llvm_variant, build_number) %}
```

Comment:
It would probably be cleaner and clearer to just use a string with the Jinja elements in the build string.